src/app/zap-templates/call-command-handler-src.zapt

```diff
@@ -100,15 +100,49 @@ EmberAfStatus emberAf{{asCamelCased name false}}Cluster{{asCamelCased side false
         {{#if (isCommandAvailable parent.side incoming outgoing commandSource name)}}
         case ZCL_{{asDelimitedMacro name}}_COMMAND_ID: {
             {{#if (zcl_command_arguments_count this.id)}}
-            uint32_t argOffset = 0;
+            uint32_t payloadOffset = cmd->payloadStartIndex;
             {{#zcl_command_arguments}}
-            {{asUnderlyingType type}} * {{asSymbol label}} = ({{asUnderlyingType type}} *)(cmd->buffer + argOffset);
+            {{#if (isNullable type isArray)}}
+            uint8_t * {{asSymbol label}};
+            {{else}}
+            {{asUnderlyingType type}} {{asSymbol label}};
+            {{/if}}
+            {{/zcl_command_arguments}}
+            {{#zcl_command_arguments}}
+            {{#if presentIf}}
```

Comment:
I would prefer the `isStrEqual` to check if the string is empty or not. Something like `{{#unless (isStrEqual presentIf '')}}`. But that's personal preference.